vmm/src/api/openapi/cloud-hypervisor.yaml

```diff
@@ -213,8 +213,10 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/VhostUserBlkConfig'
-        vsock:
-          $ref: '#/components/schemas/VsockConfig'
+        vsocks:
```

Comment:
why did you change the name `vsock` into `vsocks`?

Reply:
it is an array I felt more natural on put in plural.